ff2mpv.py

```diff
@@ -78,6 +78,15 @@ def create_cookiefile(cookies):
     return cookiefile.name
 
 
+def get_config_path(file=''):
+    path = os.path.join(os.getenv('XDG_CONFIG_HOME'), 'ff2mpv', file)
```

Comment:
I think that this should fail if XDG_CONFIG_HOME is not set, because os.getenv will return None

Reply:
Hm ok then I'll replace the if clause with a try except.

Reply:
Even better: The method allows passing a default value...